### DISCUSSION TITLE
Wrong color on React component tag?

### DISCUSSION THREAD

**Discussion by rodrigosaling** [one-dark/jetbrains-one-dark-theme#308] [General]
Hello! I don't remember when it started, but I noticed that some time ago the theme is using a "light teal" color for React component tags, as you can see below:

<img width="696" alt="image" src="https://github.com/one-dark/jetbrains-one-dark-theme/assets/2324719/7d1c5b1f-5ee1-4438-9c0d-55720fde2cfb">

I don't recall exactly what color was before, but I am pretty sure it wasn't this one, because it looks odd and because other themes also use it:

Nord
<img width="693" alt="image" src="https://github.com/one-dark/jetbrains-one-dark-theme/assets/2324719/ceb542df-95d2-43a6-8285-cdba9a92bf3d">

Monokai Pro
<img width="681" alt="image" src="https://github.com/one-dark/jetbrains-one-dark-theme/assets/2324719/b234b02b-9c62-4ab0-ae86-809e0811d5d2">

Darcula
<img width="708" alt="image" src="https://github.com/one-dark/jetbrains-one-dark-theme/assets/2324719/453742c5-985d-406d-b024-ac6b327f0b63">

But there are themes that overwrite it:

Mariana Pro
<img width="723" alt="image" src="https://github.com/one-dark/jetbrains-one-dark-theme/assets/2324719/7a5c54e9-386a-4f5f-bf2d-735f354b4d7c">

Catppuccin
<img width="689" alt="image" src="https://github.com/one-dark/jetbrains-one-dark-theme/assets/2324719/21434c40-7079-4303-9b24-c8a78a5f71c3">

----
I am using WebStorm 2023.1.3 with One Dark theme 5.8.0.

**Comment by Unthrottled** [comment]
I remember this, haven’t remembered about this when I have had time. Been a while since I have done react dev, because I noticed the same issue.

I don’t remember what it is called, something like ‘custom tag’, but you can use this https://www.jetbrains.com/help/idea/configuring-colors-and-fonts.html#jump_to_colors_and_fonts to inspect the color and potentially change it to something better. Hope that helps.

**Reply by rodrigosaling** [reply]
ohhhhhh HTML! I was looking for it in every other place than the HTML section 🤦 

<img width="1329" alt="image" src="https://github.com/one-dark/jetbrains-one-dark-theme/assets/2324719/844ec1c9-1e36-471e-bcec-6a06df958b35">

Thanks @Unthrottled !

**Reply by rodrigosaling** [reply]
🤔 Didn't work though. I will do more tests and compare against the other themes that overwrite that color to try to find a fix for this 👍

**Reply by rodrigosaling** [reply]
Ok, looks like you have to set the same color in HTML and XML sections.

[Multiple issues about this](https://youtrack.jetbrains.com/issue/WEB-52334/Color-scheme-make-it-possible-to-set-different-Component-and-Tag-colors).
